src/any-map.ts

```diff
@@ -111,7 +111,9 @@ function addSection(
   const { resolvedSources, sourcesContent: contents } = map;
 
   append(sources, resolvedSources);
-  append(names, map.names);
+  if (map.names) {
+    append(names, map.names);
```

Comment:
Instead of doing this here, we can normalize the `map` to always contain a `names` array. `map` is just a `TraceMap` instance, so we just need to update https://github.com/jridgewell/trace-mapping/blob/5ccfcfeeee9dfa3b13567bb0f95260ea32f2c269/src/trace-mapping.ts#L166